src/cephadm/cephadmlib/daemons/admin_gateway.py

```diff
@@ -105,6 +105,12 @@ def customize_container_mounts(
                 os.path.join(
                     data_dir, 'etc/nginx.conf'
                 ): '/etc/nginx/nginx.conf:Z',
+                os.path.join(
+                    data_dir, 'etc/nginx_internal_server.conf'
+                ): '/etc/nginx_internal_server.conf:Z',
+                os.path.join(
+                    data_dir, 'etc/nginx_external_server.conf'
```

Comment:
should these be added to the required_files attribute of this class?

Reply:
Done: I added all the required files and a new method to validate them